### ISSUE TITLE
Touch and multi-touch support discussion

### ISSUE THREAD

**Issue by pronvit** [mi-rack/Rack#3]
Comfortable touch operation is the first priority for miRack and obviously provides experience much closer to real hardware than mouse operation. This particular thread is for discussion of touch/multi-touch operation, originated from a [similar issue](https://github.com/VCVRack/Rack/issues/960) in VCVRack repo by @alectron.

miRack is primarily supposed to be used on ARM boards likely with a touch screen, but can be used on desktops as well. Due to inability and unwilingness of the VCVRack author to work on touch support, I may implement it in miRack for desktops as well if there’s a demand.

Anyway, the suggestions from the original thread that are going to be implemented first are

* larger hit zones for controls and ports. This is a simple change as long as larger zones of different controls don’t intersect.
* module locking, so that accidentially dragging a module background doesn’t move the module. On touch screens this will probably be the default mode, with touch and hold to move modules. 

Multi-touch is the next step once my multi-touch screen arrives and I learn how to work with multi-touch devices on Linux.

**Comment by pronvit** [comment]
Yes, it's done. I think you should be able to apply this patch to VCV code too if you're able/willing to compile yourself. I don't see another way as even if I eventually make my fork available for Windows, closed-source plugins won't work with it.

**Comment by jyg** [comment]
Hello @pronvit
I m trying to use VCV rack on microsoft surface pro 5.
The current VCV rack version is unusable.
Is it currently possible to compile mi-rack for windows ?
Or may I try to compile VCV rack version applying the patch you mentionned in first post ? (https://github.com/VCVRack/Rack/issues/960)
The problem is that I cannot figure where to apply the patch (where is the settings.json file ?)
Thanks in advance
Jean-Yves
